modules/presentation.py

```diff
@@ -0,0 +1,215 @@
+# Presentation file
+# Contains charts
+# IMPORTS:
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import re
+import nltk
+import datetime
+
+
+"""
+How to use:
+
+You have to have an array of tweets.
+1. Sort
+2. Get sentiment
+3. Plot
+4. Save plot or show
+
+First, you sort the tweets, using for example get_by_key_value or get_tweets_in_daterange
+Then you use get_sentiment 
+Then you feed the result of get_sentiment to a plotter function
+
+You can chain several filters.
+
+"""
+
+# Change here, if sentiments change
+sentiments = ["Positive", "Negative", "Uncertain"]
+
+# 1 SORTING
+# FILTERING / SORTING FUNCTIONS START
+def get_tweets_in_daterange(tweets, start_date, end_date):
+    """
+    Category: Filter function
+
+    Returns all tweets with "date" between start_date and end_date
+
+    Parameters:\n
+        tweets: array of tweets
+        start_date and end_date are datetime objects
+
+    Returns:\n
+        Filtered list of tweets within start_date and end_date
+    """
+    if !isinstance(start_date, datetime.date) or !isinstance(end_date, datetime.date):
```

Comment:
Does the negation work with **!** in python? I had som trouble with it in my own code and had to use **not** instead of **!**

Reply:
I think the error you are getting with python stems from here: https://github.com/Hold-Krykke/PythonExam/pull/6/files#diff-5b46cfda0ddaed371f52aa24ea2b0b2aR6

I will also not be able to run the program if this is comitted, as my python path is different.

Reply:
I think you are right - should that just be removed then?

Reply:
> .vscode should be gitignored.
Let's just do that before the merge then. Though the file may need to be deleted in a commit. I am still a little unsure how the gitignore works on files already on git

Reply:
Should be fine now.

Reply:
> Do you have any saved png's of the final version of the plot's we could see?
There was, but I deleted them, as per https://github.com/Hold-Krykke/PythonExam/pull/6#pullrequestreview-408962920 😅 

Get test_data_generator.py and the bottom tests, and you should be able to run the code. 

I can make a new branch, off this one, with the testing stuff, maybe?